content/guides/component-testing/quickstart-react.md

```diff
@@ -2,114 +2,140 @@
 ---
 
+Welcome! This tutorial will walk you through creating a React app and using
+Cypress Component Testing to test it. We assume you are already familiar with
+React, but if this is your first time with Cypress, that's okay; we'll walk you
+through all you need to know.
+
 <CtBetaAlert></CtBetaAlert>
 
-To follow along with this guide, you'll need a React application.
+## Getting Started
+
+### Create a React App
 
-The quickest way to get started writing component tests for React is to use one
-of the frameworks out there that can create a React project for you, like
-[Create React App](https://create-react-app.dev/), or
-[Vite](https://vitejs.dev/). In this guide, we will use Vite as it's quick to
-get up and running.
+To start off, we are going to create a new React app.
 
-To create a React project with Vite:
+We will use [Vite](https://vitejs.dev/) to create the app as it's quick to get
+up and running.
 
-1. Run the scaffold command
+To create a React project with Vite, run the following from your command prompt:
 
 ```bash
 npm create vite@latest my-awesome-app -- --template react
 ```
 
-2. Go into the directory and run `npm install`
+Go into the directory and run `npm install`:
 
 ```bash
 cd my-awesome-app
 npm install
 ```
 
-3. Add Cypress
+### Install Cypress
+
+Next, let's add Cypress to the app:
 
 ```bash
 npm install cypress -D
 ```
 
-4. Open Cypress and follow the Launchpad's prompts!
+Open Cypress:
 
 ```bash
 npx cypress open
 ```
 
-## Configuring Component Testing
+The Cypress Launchpad will now open and guide you through the configuring your
+project.
+
+### Configuring Component Testing
 
-Whenever you run Cypress for the first time in a project, the app will prompt
-you to set up either **E2E Testing** or **Component Testing**. Choose
-**Component Testing** and step through the configuration wizard.
+Whenever you run Cypress for the first time, the app will prompt you to set up
+either E2E Testing or Component Testing. Click on "Component Testing" to start
+the configuration wizard.
 
 <DocsImage 
-  src="/img/guides/component-testing/select-test-type.png" 
+  src="/img/guides/component-testing/select-test-type.jpg" 
   caption="Choose Component Testing"> </DocsImage>
 
-The Project Setup screen automatically detects your framework and bundler, which
-is React and Vite in our case. Cypress Component Testing uses your existing
-development server config to render components, helping ensure your components
-act and display in testing the same as they do in production.
+The Project Setup screen automatically detects your framework and bundler,
+which, in our case, is React and Vite. Click "Next Step" to continue.
+
+<DocsImage 
+  src="/img/guides/component-testing/project-setup-react.jpg" 
+  caption="React and Vite are automatically detected"> </DocsImage>
+
+The next screen checks that all the required dependencies are installed. All the
+items should have green checkboxes on them, indicating everything is good, so
```

Comment:
Nit pick: Maybe "green check marks"? Feels off form the UI since they technically aren't boxes.